contract/apps.py

```diff
@@ -1,5 +1,43 @@
 from django.apps import AppConfig
 
 
+MODULE_NAME = "contract"
+
+
+DEFAULT_CFG = {
+    "gql_query_contract_perms": ["152101"],
+    "gql_query_contract_admins_perms": [],
+    "gql_mutation_create_contract_perms": ["152102"],
+    "gql_mutation_update_contract_perms": ["152103"],
+    "gql_mutation_approve_ask_for_change_contract_perms": ["152108"],
+}
```

Comment:
we might have to rework that part later to wrap it in a "perms" dict add some label for the Profile page 

"perms":{"contract_perms": {"id":"152101", "label":{"fr":"Contrat", "en":"Contract"}}} ....

Reply:
Ok. We can discuss this later related to the new Profile page etc.